jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -65,6 +65,9 @@ def buildExclusions = [
   // OSX ---------------------------------------------------
   [ /^osx1010/,                       anyType,     gte(11) ],
   [ /^osx1011/,                       releaseType, lt(11)  ],
+  
+  // FreeBSD -----------------------------------------------
+  [ /^freebsd10/,                     anyType,     lt(11)  ],
```

Comment:
Wait, I think I did it the opposite way. It should be `gte(11)`, right?

Reply:
Yes, this is a list of exclusions, see:
https://github.com/nodejs/build/blob/d121160b7d38cecd98c794a2a9549b89708a4754/jenkins/scripts/VersionSelectorScript.groovy#L8-L9